magento/magento2
# Removed redundant whitespace from order options

### Description (*)

Removed redundant whitespace so that the custom option value can be copied as a whole, instead of having to remove the whitespace every time.

A gif says more than 1000 words, before situation:

![before](https://user-images.githubusercontent.com/7881219/61801140-926f3100-ae2e-11e9-9293-9bd555bce1e1.gif)

And after the fix:

![after](https://user-images.githubusercontent.com/7881219/61801145-9438f480-ae2e-11e9-8f23-cb49af6f7636.gif)


### Fixed Issues (if relevant)


### Manual testing scenarios (*)

Add a custom option to a product, long enough that it will be cut off (the dots show up) and hover over the dots. See gifs in description.


### Questions or comments


### Contribution checklist (*)
 - [] All automated tests passed successfully (all builds are green)
